### ISSUE TITLE
Update Money or other dependencies

### ISSUE THREAD

**Issue by fjg** [glebtv/mongoid_money_field#4]
It blocks the dependency tree of some app using it...

**Comment by Elyasin** [comment]
I agree. It would be great if this gem could be updated to reflect latest versions of the dependencies.
